backend/ltsa/LtsaService.cs

```diff
@@ -0,0 +1,286 @@
+        /// </summary>
+        /// <param name="pid"></param>
+        /// <returns></returns>
+        public async Task<OrderWrapper<TitleOrder>> PostTitleOrder(string titleNumber, string landTitleDistrictCode)
+        {
+            TitleOrder order = new TitleOrder(new TitleOrderParameters(titleNumber, Enum.Parse<LandTitleDistrictCode>(landTitleDistrictCode)));
+            var url = this.Options.HostUri.AppendToURL(this.Options.OrdersEndpoint);
+            return await SendAsync<OrderWrapper<TitleOrder>, OrderWrapper<TitleOrder>>(url, HttpMethod.Post, new OrderWrapper<TitleOrder>(order));
+        }
+
+        /// <summary>
+        /// Post a Parcel Info Order using the passed pid
+        /// </summary>
+        /// <param name="pid"></param>
+        /// <returns></returns>
+        public async Task<OrderWrapper<ParcelInfoOrder>> PostParcelInfoOrder(string pid)
+        {
+            ParcelInfoOrder order = new ParcelInfoOrder(new ParcelInfoOrderParameters(pid));
+            var url = this.Options.HostUri.AppendToURL(this.Options.OrdersEndpoint);
+            return await SendAsync<OrderWrapper<ParcelInfoOrder>, OrderWrapper<ParcelInfoOrder>>(url, HttpMethod.Post, new OrderWrapper<ParcelInfoOrder>(order));
+        }
+
+        /// <summary>
+        /// Post a Strata Plan Common Property order using the passed strataPlanNumber
+        /// </summary>
+        /// <param name="strataPlanNumber"></param>
+        /// <returns></returns>
+        public async Task<OrderWrapper<SpcpOrder>> PostSpcpOrder(string strataPlanNumber)
+        {
+            SpcpOrder order = new SpcpOrder(new StrataPlanCommonPropertyOrderParameters(strataPlanNumber));
+            var url = this.Options.HostUri.AppendToURL(this.Options.OrdersEndpoint);
+            return await SendAsync<OrderWrapper<SpcpOrder>, OrderWrapper<SpcpOrder>>(url, HttpMethod.Post, new OrderWrapper<SpcpOrder>(order));
+        }
+
+        /// <summary>
+        /// Retrieve all of the title summaries for the given pid, and create a title order for each one. Return all of the title orders and the parcel info order for the given pid
+        /// </summary>
+        /// <param name="pid"></param>
+        /// <returns></returns>
+        public async Task<LtsaOrders> PostLtsaFields(string pid)
+        {
+            int parcelIdentifier = ConvertPID(pid);
+            var titleSummariesTask = GetTitleSummariesAsync(parcelIdentifier);
+            var PostParcelInfoOrderTask = PostParcelInfoOrder(pid);
+            var titleSummaries = await titleSummariesTask;
+
+            ICollection<Task<OrderWrapper<TitleOrder>>> titleOrderTasks = new List<Task<OrderWrapper<TitleOrder>>>();
+            foreach (TitleSummary titleSummary in titleSummaries.TitleSummaries)
+            {
+                titleOrderTasks.Add(PostTitleOrder(titleSummary.TitleNumber, titleSummary.LandTitleDistrictCode.ToString()));
+            }
+            var titleOrders = await Task.WhenAll(titleOrderTasks);
```

Comment:
Having multiple requests, can we result in a timeout in the original request that executed these?

Reply:
potentially, but that would have to be handled on the client side would it not?